Update Lowdefy config

SamTolmay opened the pull request:
<!--
  Thanks for submitting a pull request!
  Please:
    - provide enough information so that others can review your pull request.
    - double check [the dedicated documentation available here](https://community.algolia.com/docsearch/documentation/)
    - try [to implement the recommendations](https://community.algolia.com/docsearch/documentation/docsearch/recommendations/)
    - please feature [a sitemap](https://www.sitemaps.org/), it will be the most complete source of truth for our crawling.
    - [Allow edits from maintainer](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
-->


<!--
  Explain the **motivation** for making this change.
  What existing problem does the pull request solve?
  Are there any linked issues?
  Please attach also a URL showing that you are a maintainer of the project.
-->
# Pull request motivation(s)

We have changed our documentation site and would like to reactivate Docsearch on our site. I have updated the configuration for the new site, but I am not sure that it is correct.

### What is the current behaviour?

Docsearch is currently deactivated, and the config is for a previous version of the documentation site.

### What is the expected behaviour?

Update the configuration.


##### NB: Do you want to request a **feature** or report a **bug**?


##### NB2: Any other feedback / questions ?

Do we need to get a new  API key for the Docsearch widget?

<!--
  The CI will check that the configuration is compliant with the JSON schema we have defined, please make sure the check is passed. Let us know if you do not get the issue.
-->

shortcuts commented:
Hi @SamTolmay,

I've updated the selectors of you config, but we are still not able to retrieve all the available URLs.

Would you be able to add a `sitemap.xml` so we can proceed?

(The new API key will be sent to you directly by email)

SamTolmay commented:
Hi @shortcuts

Thanks for the update to the config.
I will be able to add a sitemap in the next few days. I will update this PR with the sitemap URL once I have added it.

SamTolmay commented:
HI @shortcuts 

We have added a sitemap to our docs and I have updated the config with the URL. I see the CI is failing, but I am not sure what is wrong with the config.
